### PR TITLE
Updated security alerts tests + added Secure score types tests

### PR DESCRIPTION
**Pull request by giromm2ms**
* Added Seucre Score tests

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Security/Security.Test/Security.Test.csproj [hunk 1]
```diff
@@ -10,38 +10,12 @@
...
 
+  <!--<ItemGroup>
+    <None Update="PemFiles\*.*" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>-->
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.SecurityCenter" Version="2.1.0" />
-  </ItemGroup>
-  
-  <ItemGroup>
```
Comment (eliagrady) [inline src/Security/Security.Test/Security.Test.csproj:17]: Is this removal on purpose? why was it added to begin with?

File: src/Security/Security/Models/SecureScore/PSSecuritySecureScoreConverter.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        public static PSSecuritySecureScore ConvertToPSType(this SecureScoreItem value)
+        {
+            return new PSSecuritySecureScore
+            {
+                Id = value.Id,
+                Name = value.Name,
+                Type = value.Type,
+                DisplayName = value.DisplayName,
+                CurrentScore = value.Current,
```
Comment (eliagrady) [inline src/Security/Security/Models/SecureScore/PSSecuritySecureScoreConverter.cs:31]: validate `value.Current` is correct and not `value.CurrentScore`

File: src/Security/Security/Models/SecureScore/PSSecuritySecureScoreControlConverter.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    {
+        public static PSSecuritySecureScoreControl ConvertToPSType(this SecureScoreControlDetails value)
+        {
+            return new PSSecuritySecureScoreControl
+            {
+                Id = value.Id,
+                Name = value.Name,
+                Type = value.Type,
+                DisplayName = value.DisplayName,
+                CurrentScore = value.Current,
```
Comment (eliagrady) [inline src/Security/Security/Models/SecureScore/PSSecuritySecureScoreControlConverter.cs:31]: same

File: src/Security/Security.Test/ScenarioTests/SecurityAlertTests.ps1 [hunk 4]
```diff
@@ -28,7 +28,7 @@ Get security alerts on a resource group and its resources
 #>
 function Get-AzureRmSecurityAlert-ResourceGroupScope
 {
-	$rgName = Get-TestResourceGroupName
```
Comment (eliagrady) [inline src/Security/Security.Test/ScenarioTests/SecurityAlertTests.ps1:31]: where was `Get-TestResourceGroupName` defined? why we do not need it anymore? (didn't see a removal of it)

**Review by eliagrady — APPROVED** [review]
Looks fine, get someone from Detections team to review changes to alerts tests as well

File: src/Security/Security.Test/ScenarioTests/SecurityAlertTests.ps1 (hunk 4 above)
Comment (giromm2ms) [reply src/Security/Security.Test/ScenarioTests/SecurityAlertTests.ps1:31]: It is defined in Common.ps1
We use here Alerts Simulator resource group name instead

File: src/Security/Security.Test/Security.Test.csproj (hunk 1 above)
Comment (giromm2ms) [reply src/Security/Security.Test/Security.Test.csproj:17]: IT should not been added.
The Test.csproj should update the nuget to the latest version

**Comment by giromm2ms** [comment]
> Looks fine, get someone from Detections team to review changes to alerts tests as well

Changes were made with @yibirnba

**Comment by msJinLei** [comment]
@giromm2ms There are several test cases failed in pipeline. Could you fix them?
Here is an examples
```
2020-11-30T17:53:06.0421494Z   A total of 1 test files matched the specified pattern.
2020-11-30T17:53:11.1605463Z   [xUnit.net 00:00:03.70]     Microsoft.Azure.Commands.Security.Test.ScenarioTests.SqlInformationProtectionPolicyTests.TestErrorWhenRankIsInvalid [FAIL]
2020-11-30T17:53:11.4022645Z     X Microsoft.Azure.Commands.Security.Test.ScenarioTests.SqlInformationProtectionPolicyTests.TestErrorWhenRankIsInvalid [2s 151ms]
2020-11-30T17:53:11.4023364Z EXEC : error Message:  [D:\a\1\s\build.proj]
2020-11-30T17:53:11.4024444Z      System.Management.Automation.RuntimeException : Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4026792Z   ---- System.Management.Automation.RuntimeException : Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4027552Z     Stack Trace:
2020-11-30T17:53:11.4027908Z        at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
2020-11-30T17:53:11.4039483Z      at System.Management.Automation.PowerShell.Worker.ConstructPipelineAndDoWork(Runspace rs, Boolean performSyncInvoke)
2020-11-30T17:53:11.4039962Z      at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync)
2020-11-30T17:53:11.4040426Z      at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
2020-11-30T17:53:11.4040961Z      at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
2020-11-30T17:53:11.4041463Z      at System.Management.Automation.PowerShell.CoreInvoke[TOutput](IEnumerable input, PSDataCollection`1 output, PSInvocationSettings settings)
2020-11-30T17:53:11.4042044Z      at System.Management.Automation.PowerShell.Invoke(IEnumerable input, PSInvocationSettings settings)
2020-11-30T17:53:11.4042391Z      at System.Management.Automation.PowerShell.Invoke()
2020-11-30T17:53:11.4042877Z      at Microsoft.WindowsAzure.Commands.ScenarioTest.EnvironmentSetupHelper.RunPowerShellTest(String[] scripts) in D:\a\1\s\tools\ScenarioTest.ResourceManager\EnvironmentSetupHelper.cs:line 573
2020-11-30T17:53:11.4043691Z      at Microsoft.Azure.Commands.Security.Test.ScenarioTests.TestController.RunPowerShellTest(XunitTracingInterceptor logger, String[] scripts) in D:\a\1\s\src\Security\Security.Test\ScenarioTests\TestController.cs:line 70
2020-11-30T17:53:11.4047916Z      at Microsoft.Azure.Commands.Security.Test.ScenarioTests.SqlInformationProtectionPolicyTests.TestErrorWhenRankIsInvalid() in D:\a\1\s\src\Security\Security.Test\ScenarioTests\SqlInformationProtectionPolicyTests.cs:line 78
2020-11-30T17:53:11.4048500Z   ----- Inner Stack Trace -----
2020-11-30T17:53:11.4048714Z   
2020-11-30T17:53:11.4048901Z     Standard Output Messages:
2020-11-30T17:53:11.4049141Z    Test-ErrorWhenRankIsInvalid
2020-11-30T17:53:11.4050512Z    Caught Exception: System.Management.Automation.RuntimeException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.' ---> System.Management.Automation.RuntimeException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4051960Z       --- End of inner exception stack trace ---
2020-11-30T17:53:11.4052360Z       at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
2020-11-30T17:53:11.4052762Z       at System.Management.Automation.PowerShell.Worker.ConstructPipelineAndDoWork(Runspace rs, Boolean performSyncInvoke)
2020-11-30T17:53:11.4053216Z       at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync)
2020-11-30T17:53:11.4053702Z       at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
2020-11-30T17:53:11.4054554Z       at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
2020-11-30T17:53:11.4055337Z       at System.Management.Automation.PowerShell.CoreInvoke[TOutput](IEnumerable input, PSDataCollection`1 output, PSInvocationSettings settings)
2020-11-30T17:53:11.4055999Z       at System.Management.Automation.PowerShell.Invoke(IEnumerable input, PSInvocationSettings settings)
2020-11-30T17:53:11.4056482Z       at System.Management.Automation.PowerShell.Invoke()
2020-11-30T17:53:11.4057164Z       at Microsoft.WindowsAzure.Commands.ScenarioTest.EnvironmentSetupHelper.RunPowerShellTest(String[] scripts) in D:\a\1\s\tools\ScenarioTest.ResourceManager\EnvironmentSetupHelper.cs:line 573
2020-11-30T17:53:11.4058593Z    Message: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4060180Z EXEC : PowerShell error Record: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.' [D:\a\1\s\build.proj]
2020-11-30T17:53:11.4061874Z   Exception:System.Management.Automation.RuntimeException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4062702Z   Details:
2020-11-30T17:53:11.4063206Z   Script Stack Trace: at Assert-Throws, D:\a\1\s\src\Security\Security.Test\bin\Debug\netcoreapp2.1\Assert.ps1: line 42
2020-11-30T17:53:11.4063884Z    at Test-ErrorWhenRankIsInvalid, D:\a\1\s\src\Security\Security.Test\bin\Debug\netcoreapp2.1\ScenarioTests\SqlInformationProtectionPolicyTests.ps1: line 92
2020-11-30T17:53:11.4064644Z    at <ScriptBlock>, <No file>: line 1
2020-11-30T17:53:11.4065479Z   : Target: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4066261Z   
2020-11-30T17:53:11.4067107Z    Caught Exception: System.Management.Automation.RuntimeException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4068613Z    Message: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4070203Z EXEC : PowerShell error Record: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.' [D:\a\1\s\build.proj]
2020-11-30T17:53:11.4071785Z   Exception:System.Management.Automation.ParentContainsErrorRecordException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
2020-11-30T17:53:11.4072655Z   Details:
2020-11-30T17:53:11.4072956Z   Script Stack Trace: 
2020-11-30T17:53:11.4073322Z   : Target: 
```

**Comment by giromm2ms** [comment]
> @giromm2ms There are several test cases failed in pipeline. Could you fix them?
> Here is an examples
> 
> ```
> 2020-11-30T17:53:06.0421494Z   A total of 1 test files matched the specified pattern.
> 2020-11-30T17:53:11.1605463Z   [xUnit.net 00:00:03.70]     Microsoft.Azure.Commands.Security.Test.ScenarioTests.SqlInformationProtectionPolicyTests.TestErrorWhenRankIsInvalid [FAIL]
> 2020-11-30T17:53:11.4022645Z     X Microsoft.Azure.Commands.Security.Test.ScenarioTests.SqlInformationProtectionPolicyTests.TestErrorWhenRankIsInvalid [2s 151ms]
> 2020-11-30T17:53:11.4023364Z EXEC : error Message:  [D:\a\1\s\build.proj]
> 2020-11-30T17:53:11.4024444Z      System.Management.Automation.RuntimeException : Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4026792Z   ---- System.Management.Automation.RuntimeException : Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4027552Z     Stack Trace:
> 2020-11-30T17:53:11.4027908Z        at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
> 2020-11-30T17:53:11.4039483Z      at System.Management.Automation.PowerShell.Worker.ConstructPipelineAndDoWork(Runspace rs, Boolean performSyncInvoke)
> 2020-11-30T17:53:11.4039962Z      at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync)
> 2020-11-30T17:53:11.4040426Z      at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
> 2020-11-30T17:53:11.4040961Z      at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
> 2020-11-30T17:53:11.4041463Z      at System.Management.Automation.PowerShell.CoreInvoke[TOutput](IEnumerable input, PSDataCollection`1 output, PSInvocationSettings settings)
> 2020-11-30T17:53:11.4042044Z      at System.Management.Automation.PowerShell.Invoke(IEnumerable input, PSInvocationSettings settings)
> 2020-11-30T17:53:11.4042391Z      at System.Management.Automation.PowerShell.Invoke()
> 2020-11-30T17:53:11.4042877Z      at Microsoft.WindowsAzure.Commands.ScenarioTest.EnvironmentSetupHelper.RunPowerShellTest(String[] scripts) in D:\a\1\s\tools\ScenarioTest.ResourceManager\EnvironmentSetupHelper.cs:line 573
> 2020-11-30T17:53:11.4043691Z      at Microsoft.Azure.Commands.Security.Test.ScenarioTests.TestController.RunPowerShellTest(XunitTracingInterceptor logger, String[] scripts) in D:\a\1\s\src\Security\Security.Test\ScenarioTests\TestController.cs:line 70
> 2020-11-30T17:53:11.4047916Z      at Microsoft.Azure.Commands.Security.Test.ScenarioTests.SqlInformationProtectionPolicyTests.TestErrorWhenRankIsInvalid() in D:\a\1\s\src\Security\Security.Test\ScenarioTests\SqlInformationProtectionPolicyTests.cs:line 78
> 2020-11-30T17:53:11.4048500Z   ----- Inner Stack Trace -----
> 2020-11-30T17:53:11.4048714Z   
> 2020-11-30T17:53:11.4048901Z     Standard Output Messages:
> 2020-11-30T17:53:11.4049141Z    Test-ErrorWhenRankIsInvalid
> 2020-11-30T17:53:11.4050512Z    Caught Exception: System.Management.Automation.RuntimeException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.' ---> System.Management.Automation.RuntimeException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4051960Z       --- End of inner exception stack trace ---
> 2020-11-30T17:53:11.4052360Z       at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
> 2020-11-30T17:53:11.4052762Z       at System.Management.Automation.PowerShell.Worker.ConstructPipelineAndDoWork(Runspace rs, Boolean performSyncInvoke)
> 2020-11-30T17:53:11.4053216Z       at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync)
> 2020-11-30T17:53:11.4053702Z       at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
> 2020-11-30T17:53:11.4054554Z       at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
> 2020-11-30T17:53:11.4055337Z       at System.Management.Automation.PowerShell.CoreInvoke[TOutput](IEnumerable input, PSDataCollection`1 output, PSInvocationSettings settings)
> 2020-11-30T17:53:11.4055999Z       at System.Management.Automation.PowerShell.Invoke(IEnumerable input, PSInvocationSettings settings)
> 2020-11-30T17:53:11.4056482Z       at System.Management.Automation.PowerShell.Invoke()
> 2020-11-30T17:53:11.4057164Z       at Microsoft.WindowsAzure.Commands.ScenarioTest.EnvironmentSetupHelper.RunPowerShellTest(String[] scripts) in D:\a\1\s\tools\ScenarioTest.ResourceManager\EnvironmentSetupHelper.cs:line 573
> 2020-11-30T17:53:11.4058593Z    Message: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4060180Z EXEC : PowerShell error Record: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.' [D:\a\1\s\build.proj]
> 2020-11-30T17:53:11.4061874Z   Exception:System.Management.Automation.RuntimeException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4062702Z   Details:
> 2020-11-30T17:53:11.4063206Z   Script Stack Trace: at Assert-Throws, D:\a\1\s\src\Security\Security.Test\bin\Debug\netcoreapp2.1\Assert.ps1: line 42
> 2020-11-30T17:53:11.4063884Z    at Test-ErrorWhenRankIsInvalid, D:\a\1\s\src\Security\Security.Test\bin\Debug\netcoreapp2.1\ScenarioTests\SqlInformationProtectionPolicyTests.ps1: line 92
> 2020-11-30T17:53:11.4064644Z    at <ScriptBlock>, <No file>: line 1
> 2020-11-30T17:53:11.4065479Z   : Target: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4066261Z   
> 2020-11-30T17:53:11.4067107Z    Caught Exception: System.Management.Automation.RuntimeException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4068613Z    Message: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4070203Z EXEC : PowerShell error Record: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.' [D:\a\1\s\build.proj]
> 2020-11-30T17:53:11.4071785Z   Exception:System.Management.Automation.ParentContainsErrorRecordException: Expected exception not received: 'Error converting value "Non" to type 'System.Nullable`1[Microsoft.Azure.Commands.SecurityCenter.Models.SqlInformationProtectionPolicy.PSSqlInformationProtectionRank]'. Path 'Labels.50e58766-ab53-4846-be8a-35e0bb87723e.Rank', line 6, position 19.' the actual message is 'The provided file SqlInformationProtectionPolicies\InvalidRank.json does not exist.'
> 2020-11-30T17:53:11.4072655Z   Details:
> 2020-11-30T17:53:11.4072956Z   Script Stack Trace: 
> 2020-11-30T17:53:11.4073322Z   : Target: 
> ```
Sure. I'll ask the relevant teams to fix their tests.
Hopefully we will be prepared by end of day

**Comment by giromm2ms** [comment]
@msJinLei please review as tests are ok now

File: src/Security/Security/Cmdlets/Alerts/GetAlerts.cs [hunk 5]
```diff
@@ -19,11 +19,12 @@
...
 using Microsoft.Azure.Commands.Security.Models.Alerts;
 using Microsoft.Azure.Commands.SecurityCenter.Common;
-using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
-using Microsoft.Rest.Azure;
+using Microsoft.Azure.Commands.SecurityCenter.Models.Alerts;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Security.Cmdlets.Alerts
 {
+    [CmdletOutputBreakingChange(typeof(PSSecurityAlert), ReplacementCmdletOutputTypeName = "PSSecurityAlertV3")]
```
Comment (msJinLei) [inline src/Security/Security/Cmdlets/Alerts/GetAlerts.cs:27]: This is breaking change notification not breaking changes

**Review by msJinLei — APPROVED** [review]
What you added is breaking change notification rather than breaking change. I just modify your change log. We should use correct phase, or it will cause misunderstanding. 

I notice you add cmdlets in your PR. For new cmdlets, we require a design review before PR https://github.com/Azure/azure-powershell-cmdlet-review-pr. But since the cmdlets in your PR is simple, we just approve it. Please don't forget mention design review issue id in your PR with new cmdlets.

**Comment by giromm2ms** [comment]
@msJinLei ,
Thanks for thd assistance and the approval !

Your comments will be taken into account on next PRs.

 Can this one enter milestone 179 ibstead of 180?

**Comment by msJinLei** [comment]
> @msJinLei ,
> Thanks for thd assistance and the approval !
> 
> Your comments will be taken into account on next PRs.
> 
>  Can this one enter milestone 179 ibstead of 180?

I am afraid it cannot because it is already merged to master rather than release branch.
